styles.css

```diff
@@ -1,15 +1,36 @@
 body {
-  font-family: 'Roboto', sans-Serif;
+  font-family: 'Roboto', sans-serif;
   margin: 0;
 }
 
 .logo {
-  font-family: 'PT Serif', serif;
+  font: 24px/31px pt_serif, arial, helvetica, sans-serif;
   padding: 40px 0;
+  padding-left: 44px;
+  position: relative;
 }
 
-.main-content__title {
-  font-family: 'PT Serif', serif;
+.logo__circle {
+  width: 28px;
+  height: 28px;
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  border-radius: 50%;
+  opacity: .7;
+}
+
+.logo__circle_color_blue {
+  left: 8px;
+  top: 56%;
+  background: #c8dbfb;
+}
+
+.logo__circle_color_dark-blue { background: #0056ec; }
+
+.main-content__title { 
```

Comment:
Codacy found an issue: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6170621)